master/buildbot/steps/source/git.py

```diff
@@ -45,35 +45,64 @@ def isTrueOrIsExactlyZero(v):
 
 git_describe_flags = [
     # on or off
-    ('all', lambda v: ['--all'] if v else None),
-    ('always', lambda v: ['--always'] if v else None),
-    ('contains', lambda v: ['--contains'] if v else None),
-    ('debug', lambda v: ['--debug'] if v else None),
-    ('long', lambda v: ['--long'] if v else None),
-    ('exact-match', lambda v: ['--exact-match'] if v else None),
-    ('tags', lambda v: ['--tags'] if v else None),
+    ("all", lambda v: ["--all"] if v else None),
+    ("always", lambda v: ["--always"] if v else None),
+    ("contains", lambda v: ["--contains"] if v else None),
+    ("debug", lambda v: ["--debug"] if v else None),
+    ("long", lambda v: ["--long"] if v else None),
+    ("exact-match", lambda v: ["--exact-match"] if v else None),
+    ("tags", lambda v: ["--tags"] if v else None),
     # string parameter
-    ('match', lambda v: ['--match', v] if v else None),
+    ("match", lambda v: ["--match", v] if v else None),
     # numeric parameter
-    ('abbrev', lambda v: ['--abbrev={}'.format(v)]
-     if isTrueOrIsExactlyZero(v) else None),
-    ('candidates', lambda v: ['--candidates={}'.format(v)] if isTrueOrIsExactlyZero(v) else None),
+    (
+        "abbrev",
+        lambda v: ["--abbrev={}".format(v)] if isTrueOrIsExactlyZero(v) else None,
+    ),
+    (
+        "candidates",
+        lambda v: ["--candidates={}".format(v)] if isTrueOrIsExactlyZero(v) else None,
+    ),
     # optional string parameter
-    ('dirty', lambda v: ['--dirty'] if (v is True or v == '') else None),
-    ('dirty', lambda v: ['--dirty={}'.format(v)] if (v and v is not True) else None),
+    ("dirty", lambda v: ["--dirty"] if (v is True or v == "") else None),
+    ("dirty", lambda v: ["--dirty={}".format(v)] if (v and v is not True) else None),
 ]
 
 
 class Git(Source, GitStepMixin):
 
-    name = 'git'
-    renderables = ["repourl", "reference", "branch",
-                   "codebase", "mode", "method", "origin"]
-
-    def __init__(self, repourl=None, branch='HEAD', mode='incremental', method=None,
-                 reference=None, submodules=False, shallow=False, progress=True, retryFetch=False,
-                 clobberOnFailure=False, getDescription=False, config=None,
-                 origin=None, sshPrivateKey=None, sshHostKey=None, sshKnownHosts=None, **kwargs):
+    name = "git"
+    renderables = [
+        "repourl",
+        "reference",
+        "branch",
+        "codebase",
+        "mode",
+        "method",
+        "origin",
+    ]
+
+    def __init__(
```

Comment:
Could we keep the style in line with the rest of the codebase? That is, indenting at the opening parentheses.